src/Features/Admin/VendorList/index.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+
+import { TableCell, TableRow } from '@material-ui/core';
+
+import CustomTable from '../../../components/CustomTable';
+
+function VendorList() {
+  const tableData = [
```

Comment:
this should be the mock data. .  and mock data has its own directory structure for mock calls. Handle these call through epic